### ISSUE TITLE
A wrong interactive about Urza's Saga and Echoing Deeps

### ISSUE THREAD

**Issue by Greasefang** [magefree/mage#12475]
I'm playing Scale cards in Modern, and i found that when i play Echoing Deeps as a copy of Urza's Saga in my graveyard, Deeps will enters the battlefield with 2 lore counters instead of 1, and abilities of Chapter I and Chapter II triggers each twice. 
This issue makes me have 1 less construct token, weaks my army, makes me lose the game. #11826

**Comment by alexander-novo** [comment]
[[Echoing Deeps]]
[[Urza’s Saga]]

**Comment by alexander-novo** [comment]
Presumably you had an effect that increases the amount of lore counters put on things? What is the incorrect interaction here?

**Comment by ssk97** [comment]
> Presumably you had an effect that increases the amount of lore counters put on things? What is the incorrect interaction here?

No, testing with just those two cards and nothing else has it enter tapped with 2 counters and with 4 triggers going on the stack with the name of "Echoing Deeps" still on the triggers. Initial investigation suggest that there's something going wrong with the ETB replacement effect that's causing it to be applied twice, don't have the time to go further right now.

**Comment by alexander-novo** [comment]
![image](https://github.com/magefree/mage/assets/11657716/81e98b78-2255-493e-989a-9f4652313b40)

Can confirm

**Comment by alexander-novo** [comment]
There's something wrong with how the copy effect is applied. When I attempted to copy the copied land with [[Vesuva]], it asked me if I want to use Echoing Deeps' ability again, but it should have just become another copy of Urza's Saga
